### DISCUSSION TITLE
How do I / should I uninstall?

### DISCUSSION THREAD

**Discussion by dustbro** [netdata/netdata#13988] [Q&A]
Hello all.
I've installed netdata with the command `yum install netdata-ui` and all is working as expected. I recently created an account at app.netdata.cloud and tried to claim the node running the script provided by Netdata, but ran in to this error:
```
WARNING  Found an existing netdata install at /, but could not determine the install type. Usually this means you installed Netdata through your distribution’s regular package repositories or some other unsupported method.

Attempting to update an existing install is not officially supported. It may work, but it also might break your system. If you just want to claim this install, you should re-run this command with the --claim-only option instead.
```
I figured I could just uninstall and then reinstall using the netdata script instead of `yum`. Only issue is, I can't figure out how to do this. 
Could someone help me out here with uninstalling? or maybe how to use the --claim-only option?

**Comment by MrZammler** [comment]
Hi @dustbro 

Can you try running the script as provided by the UI in netdata cloud, and pass extra the `--claim-only` option ?

e.g. `wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --claim-token XXXXXXXXXXXXXXX --claim-url https://app.netdata.cloud --claim-only`.

You should, yes, be able to uninstall with `yum` then use the kickstart script to install again. But you can first try the command above.

**Reply by dustbro** [reply]
When I try --claim-only, I get this error:
```
 --- Attempting to claim existing install at /. --- 
 --- Attempting to claim agent to https://app.netdata.cloud --- 
 --- Attempting to claim agent to https://app.netdata.cloud --- 
The following non-fatal warnings or errors were encountered:

  - Found an existing netdata install at /, but could not determine the install type. Usually this means you installed Netdata through your distribution’s regular package repositories or some other unsupported method.
  - Not updating existing install at /.

 ABORTED  Unable to find usable claiming script. Reinstalling Netdata may resolve this.
```

What's the command for yum to uninstall?

**Reply by slrslr** [reply]
@dustbro what may help:
Debian/Ubuntu: sudo apt purge netdata*
CentOS/Fedora may work: yum remove netdata*;dnf remove netdata*
